### PR TITLE
Activity log: Add rewind toggle

### LINKED ISSUE #14676: Activity Log/Rewind: need a way to enable/disable Rewind feature on a site.
For now, by default, the Rewind engine is not enabled for VP sites, and they need to be flagged on the VP side.  There are endpoints in place already to do this, but we need a UI to enable it.  

There is a super basic UI in place in the master branch https://github.com/Automattic/wp-calypso/pull/10862/files#diff-2fa44179e7dbe068d8ec609d8b69d699R1, which we might just want to use for the internal MVP for now, until we flesh out how it will work in the future.  

That implementation in the master branch is not only the UI, but is a functional component that hits real endpoints.  However, it relies on most of the data pieces to be merged for it to work.

### PR DESCRIPTION
**Pull request by sirreal**
Adds rewind toggle. Currently the toggle is always in "deactivated," "NotToggling" state. That will need to be corrected selecters land in #14900 ✅ 

Includes #14976 which makes the toggles behave correctly and update after toggling.

To test:
1. Vist one
   * https://calypso.live/stats/activity?branch=add/activitylog-rewind-toggle
   * http://calypso.localhost:3000/stats/activity
1. Pick a Jetpack site
1. The toggle should be disabled until sites are loaded (requires `siteId`).
1. Click the toggle.
1. Ensure you see an activate request fire in network.
1. Try toggling on and off.

~Note: `REWIND_ACTIVATE_SUCCESS` or `REWIND_DEACTIVATE_SUCCESS` is dispatched when the request comes back, reducer should be updated to correctly adjust state.~ Implemented in included #14976

All toggles states:

toggles

Fixes #14676.

### CODE REVIEW

**Comment by dereksmart** [comment]
Sorry for not thinking about this until now, but I'm not sure we need a `deactivate` button at all.  Rewind will be enabled for all sites when this is launched, and we only need a way for people to activate it for testing.  I would suggest that if/when rewind is active, there is no button or toggle.

**Comment by keoshi** [comment]
I agree with @dereksmart — don't think we need a `deactivate` button, and I don't see the use case for it. If we do have the need for it in the future, shouldn't the styling swap between them?

- Activate: `is-primary` (blue)
- Deactivate: regular styled button (white)

File: client/my-sites/stats/activity-log/index.jsx [hunk 1]
```diff
@@ -381,9 +385,10 @@ export default connect(
 			siteId,
 			slug: getSiteSlug( state, siteId ),
 			rewindStatusError: getRewindStatusError( state, siteId ),
+			isRewindActive: isRewindActiveSelector( state, siteId ),
 
 			// FIXME: Testing only
-			isPressable: get( state.activityLog.rewindStatus, [ siteId, 'isPressable' ], false ),
+			isPressable: true || get( state.activityLog.rewindStatus, [ siteId, 'isPressable' ], false ),
```
Comment (seear) [inline client/my-sites/stats/activity-log/index.jsx]: Don't forget to remove this.

**Comment by dereksmart** [comment]
gah, needs another rebase but LGTM too

File: client/my-sites/stats/activity-log/index.jsx (hunk 1 above)
Comment (sirreal) [inline client/my-sites/stats/activity-log/index.jsx]: 😬  Thanks! I knew I'd left one of these time bombs...

**Comment by sirreal** [comment]
Re-re-rebased and I cleaned up that `true || ...`. Should be ready to merge.

**Pull request merged**